flink-runtime-web/web-dashboard/src/app/pages/job/overview/accumulators/job-overview-drawer-accumulators.component.html

```diff
@@ -37,7 +37,10 @@
         <tr *ngFor="let accumulator of listOfAccumulator; trackBy:trackAccumulatorBy;">
           <td>{{ accumulator.name }}</td>
           <td>{{ accumulator.type }}</td>
-          <td>{{ accumulator.value }}</td>
+          <td *ngIf="isFinite(accumulator.value); else nonNumeric">{{ accumulator.value | number:'1.0-0' }}</td>
```

Comment:
I'm not sure, but is the `{{accumulator.value}}` is always integer? if not we should not make it round to intergerwith `number:'1.0-0'`,`number:'1.0-3'` is better